### DISCUSSION TITLE
exporting gcode

### DISCUSSION THREAD

**Discussion by 1GR3** [t-oster/VisiCut#643] [Ideas]
Hello!
Is there a way to use VisiCut to export gcode files which can then be used on StepCraft via UCCNC?
(The way the Ultimaker Cura is used for 3D printing)

**Comment by mgmax** [comment]
There is a "Generic GCode" driver, you can try to use this

**Reply by mgmax** [reply]
@1GR3 did it work? Please give feedback, and feel free to make your configuration public if you have it working.
